app/models/access.rb

```diff
@@ -1,17 +1,21 @@
 class Access < ApplicationRecord
   ALLOWED_RESOURCES = %w[Organization FacilityGroup Facility].freeze
   ACTION_TO_ROLE = {
-    view: [:manager, :viewer],
+    manage_overdue_list: [:manager, :health_care_worker, :call_center_worker],
+    view_reports: [:manager, :health_care_worker, :health_administrator],
+    view_pii: [:manager, :health_care_worker],
     manage: [:manager]
   }
 
   belongs_to :user
   belongs_to :resource, polymorphic: true, optional: true
 
   enum role: {
-    viewer: "viewer",
+    super_admin: "super_admin",
     manager: "manager",
-    super_admin: "super_admin"
+    health_care_worker: "health_care_worker",
+    health_administrator: "health_administrator",
+    call_center_worker: "call_center_worker"
```

Comment:
Following up from [https://simpledotorg.slack.com/archives/CK95QFJA2/p1596812158352700](here), I think this is a good place to start:
* manager
* viewer
* viewer_reports_only
* call_center

Reply:
I'd like to be explicit about the `viewer` role having access to reports + pii. How about `all_viewer` or `viewer_all`?